### ISSUE TITLE
Consistent way of writing

### ISSUE THREAD

**Issue by marcthiele** [sebsel/seblog-kirby-webmentions#1]
Hey,

first of all: lovely work. Thanks a lot.

I just thought, that it might be nice to be consistent in who you write the headlines for what mention it is. Either all lower case or all upper case. Easy to solve on my end of course, but I thought you maybe want to have a consistent way of doing this in the repo already. I attached screenshots of what I mean. Maybe you also had a reason for doing it this way … ;)

![screenshot 2017-02-07 13 49 27](https://cloud.githubusercontent.com/assets/293397/22691804/63e10b20-ed3c-11e6-9138-f7e5e1aa4613.png)

![screenshot 2017-02-07 13 50 54](https://cloud.githubusercontent.com/assets/293397/22691831/82beac1e-ed3c-11e6-8dcf-58ae0072857d.png)

Cheers,

/marc

**Comment by sebsel** [comment]
Oh, I actually don't use that part of the code! This comes from the original [kirby-webmentions](https://github.com/bastianallgeier/kirby-webmentions) plugin by Bastian Allgeier. I have my own snippets in `site/snippets/webmentions`, with Dutch translations of the above, but also with a few changes regarding presentation.

See examples here:
https://indieweb.org/facepile#Sebastiaan_Andeweg

I don't know if I want to ship the plugin with those icons, but I will make some changes in the default snippets to solve this better :)

(Also, make sure you got the latest version of either my seblog-kirby-webmentions or Bastian's kirby-webmentions, because it fixes a nasty vulnerability. Haven't blogged about it yet, I will soon.)

**Comment by marcthiele** [comment]
Ah ok. Thanks for clarifying. I don't think Bastian is maintaining the plugin right now. Will look deeper into your other things as well. The toolkit looks very nice.

**Comment by sebsel** [comment]
Nope, that's why I started my own fork. First just for myself, but I think it's nice to have a version that is useful to others also. So: thanks for using it! Feedback is appreciated :)

**Comment by marcthiele** [comment]
Great. Again: your work is highly appreciated ;)

I still ave a lot to integrate such as the possibility to reply from within my blog. I wish I could attend the IndieWebCamp I organise in Düsseldorf once, to find the time. Are you coming?

**Comment by sebsel** [comment]
Well, if you don't mind doing some manual work, you can actually just add: 
```html
<div class="h-entry">
In reply to <a href="https://seblog.nl/2017/02/07/3/reply" class="u-in-reply-to">Sebastiaan</a>
<p class="p-content p-name">... :)</p>
</div>
```
... to the `text` field of your entry, and a `<a href="/" class="u-url" rel="author"></a>` to the h-card in your footer.
The plugin should pick that up and send a mention. Or real manual: you can put the URL in the form underneath my reply, then my site sends the mention to itself by itself :)

I use a separate field for it, because it makes sense to do so in Kirby, but it's not required!

I plan on coming to Düsseldorf, really looking forward to that!
